app/src/main/java/com/incwelltechnology/lms/base/BaseActivity.kt

```diff
@@ -0,0 +1,16 @@
+package com.incwelltechnology.lms.base
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+
+abstract class BaseActivity<B : ViewDataBinding> : AppCompatActivity() {
+    lateinit var dataBinding: B
+    override fun onCreate(savedInstanceState: Bundle?) {
```

Comment:
How do we write all the required codes which we used to write within onCreate() function in the other activity which extends this base activity?